dotnet-desktop-guide/framework/winforms/windows-forms-data-binding.md

```diff
@@ -1,7 +1,8 @@
 ---
-title: Data Binding
+title: Data Binding in Windows Forms
```

Comment:
Files in the `framework/winforms` area automatically receive this suffix via the docfx file https://github.com/dotnet/docs-desktop/blob/main/dotnet-desktop-guide/docfx.json#L142

```suggestion
title: Data Binding
```

Reply:
@adegeo I was having a bad time with my MSFT account credentials. Everything is up and working fine now. Regarding this suggestion, if we change the title to "Data Binding" then it would be same as the title in file "**dotnet-desktop-guide/framework/winforms/data-binding-and-windows-forms.md**." Can we change windows-forms-data-binding.md's title to "Windows Forms and Data Binding"? Please suggest.

Reply:
Hi @poojapoojari welcome back! :) I would change the title of the other article to `Data Binding Overview` and leave this one `Data Binding`. This page's existence is an unfortunate artifact from the old MSDN docs design of a decade ago where we had these weird landing pages that just linked to the child content. This page doesn't have much value. So the other article is better suited to win the "Overview" name and leave this one as it is.

Reply:
@adegeo As suggested, I have updated the title for both the files. Please do have a look. :)